docs/industrial_optimization/productizing_apps.rst

```diff
@@ -0,0 +1,171 @@
+    you may want to upload it separately.
+
+#.  (Optional) Although polling the hybrid solver for completion of hybrid
+    problems is automatically performed by the Ocean SDK, you may want to
+    implement polling yourself; if you do so, you must explicitly download each
+    problem's results (i.e., the answer).
+
+#.  Handle solver errors as follows:
+
+    *   You do not need to retry idempotent requests because, by default, the
+        Ocean SDK retries them.
+
+    *   You should handle the errors raised when falling back to another
+        QPU solver is recommended or required.
+
+    *   You may retry on timeouts to non-idempotent endpoints. For example, you
+        could retry a problem submission, being aware that duplicate problems
+        could be submitted.
+
+    *   It is highly recommended that you never automatically retry on
+        authentication and authorization errors as well as HTTP 5xx errors.
+        Although an exception could be made for retrying on these errors while
+        polling, you should ensure that you back off or stop retrying during
+        long solver outages.
+    
+#.  Post-process the problem results, which typically includes the following:
+
+    *   Unpacking the problem results.
+
+    *   Interpreting and converting the problem results into a form suitable for
+        the appropriate business system and users.
+
+    *   Sending notifications to business systems and users.
+    
+A real-world example of an enterprise-level hybrid application is
+Pattison Food Group's
+`production application <https://www.dwavequantum.com/resources/application/e-comm-driver-auto-scheduling-pattison-food-group>`_,
+which uses D-Wave's hybrid solvers to optimize the weekly assignment
+of drivers' work assignments, taking into account various constraints, such as
+seniority and the required number of drivers for each shift.
+
+.. _quantum_apps_guidelines:
+
+Application Guidelines
+======================
+
+An application should be capable of handling large-scale, long-running
+hybrid problems in a heterogeneous environment typical of many enterprise-level
+businesses.
+
+.. _quantum_apps_performance_scaling:
+
+Performance and Scaling
+-----------------------
+
+Some performance and scaling considerations are the following:
+
+*   Desired throughput (i.e., solved problems per second). Also take into
+    account concurrency limits to provision your workloads effectively.
+
+*   Internet connection latency and bandwidth.
+
+*   Size of problems, which can range from about 10 MB to greater than 1 GB.
+
+*   Long and asynchronous jobs; hybrid problems can be configured to run for a
+    few seconds or several hours.
+
+*   CPU and memory requirements for hardware and virtual compute: How you model
+    your problems can impact these requirements.
+
+You should also keep up to date with Ocean SDK releases to help ensure the best
+performance and take advantage of the newest features.
```

Comment:
These seem split into two categories: ensuring hardware and configuration of the production environment is good and maintaining the application code. Worth splitting into two intro sentences so it's clear where and what actions the users needs to take.

Something like, 

- "Ensure the hardware and it's configuration in your production environment enables your application to perform and scale as needed:"
- "Maintain the code of your hybrid application: "